TS2300: Duplicate identifier 'onCursorActivity'.

为什么要定义两个onCursorActivity？从而导致Duplicate提示？

Here is the error log:

```bash
node_modules/@uiw/react-codemirror/typings/index.d.ts:54:3 - error TS2300: Duplicate identifier 'onCursorActivity'.

54   onCursorActivity?: (instance: CodeMirror.Editor) => void;
     ~~~~~~~~~~~~~~~~

node_modules/@uiw/react-codemirror/typings/index.d.ts:94:3 - error TS2300: Duplicate identifier 'onCursorActivity'.

94   onCursorActivity?(instance: CodeMirror.Editor): void;
     ~~~~~~~~~~~~~~~~

[9:20:08 AM] Found 2 errors. Watching for file changes.
```

--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -89,10 +89,6 @@ export interface IReactCodemirror extends IDOMEvent {
    * */
   onOverwriteToggle?: (instance: CodeMirror.Editor, overwrite: boolean) => void;
   /**
-   * Fired whenever the cursor or selection in this document changes.
-   * */
-  onCursorActivity?(instance: CodeMirror.Editor): void;
-  /**
    * Will be fired when the line object is deleted. A line object is associated with the start of the line.
    * Mostly useful when you need to find out when your gutter markers on a given line are removed.
    * */

